source/containers.tex

```diff
@@ -8345,11 +8346,11 @@
   template <class T, class Container = deque<T>>
   class queue {
   public:
-    typedef typename Container::value_type            value_type;
-    typedef typename Container::reference             reference;
-    typedef typename Container::const_reference       const_reference;
-    typedef typename Container::size_type             size_type;
-    typedef          Container                        container_type;
+    using value_type      = typename Container::value_type;
+    using reference       = typename Container::reference;
+    using const_reference = typename Container::const_reference;
+    using size_type       = typename Container::size_type;
+    using container_type  =          Container;
```

Comment:
I like my 2-dimensional code :)  But defer to the actual project editor on style choice - will update this tonight.